### PR TITLE
I have updated the first comment to make it more amazing.

### PR DESCRIPTION
**Pull request by bassnate**
Yeah!

### CODE REVIEW

**Comment by gtinch** [comment]
Sweet.

**Comment by gtinch** [comment]
In real time!

File: src/org/usfirst/frc2083/TeamBlitzRobot2015/Robot.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // Java from RobotBuilder. Do not put any code or make any change in
 // the blocks indicating autogenerated code or it will be lost on an
 // update. Deleting the comments indicating the section will prevent
-// it from being updated in the future.
+// it from being updated in the future. Nate Craig made an edit here.
```
Comment (dpond) [inline src/org/usfirst/frc2083/TeamBlitzRobot2015/Robot.java:9]: That was a really stupid change.

File: src/org/usfirst/frc2083/TeamBlitzRobot2015/Robot.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 // Java from RobotBuilder. Do not put any code or make any change in
 // the blocks indicating autogenerated code or it will be lost on an
 // update. Deleting the comments indicating the section will prevent
-// it from being updated in the future.
```
Comment (gtinch) [inline src/org/usfirst/frc2083/TeamBlitzRobot2015/Robot.java]: Tottally Stuepipd

**Pull request merged**